### PR TITLE
PIP-1201-talon-v5.0

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -96,10 +96,8 @@ RUN git clone --branch 1.9 --single-branch https://github.com/samtools/samtools.
     git clone --branch 1.9 --single-branch git://github.com/samtools/htslib.git && \
     cd samtools && make && make install && cd ../ && rm -rf samtools* htslib*
 
-# Install TALON 4.2
-RUN git clone -b 'v4.2' --single-branch https://github.com/dewyman/TALON.git
-RUN chmod 755 TALON/initialize_talon_database.py TALON/talon.py TALON/post-TALON_tools/create_abundance_file_from_database.py TALON/post-TALON_tools/create_GTF_from_database.py
-ENV PATH="/software/TALON:/software/TALON/post-TALON_tools:${PATH}"
+# Install TALON v5.0
+RUN git clone -b 'v5.0' --single-branch https://github.com/mortazavilab/TALON.git && cd TALON && python3.7 -m pip install .
```
Comment (paul-sud) [inline Dockerfile:100]: It looks like you could `rm -rf` the TALON repo after you pip install it

**Review by paul-sud — CHANGES_REQUESTED** [review]
One comment in the Dockerfile

File: Dockerfile (hunk 1 above)
Comment (ottojolanki) [reply Dockerfile:100]: Yep. Good catch.

Comment (paul-sud) [reply Dockerfile:100]: Great, will approve after tests pass

**Review by paul-sud — APPROVED** [review]
LGTM

**Pull request merged**